docs/syntax/pie.md

```diff
@@ -37,6 +37,11 @@ Drawing a pie chart is really simple in mermaid.
   - Followed by `:` colon as separator
   - Followed by `positive numeric value` (supported up to two decimal places)
 
+**Note:**
+
+> Pie chart values must be **positive numbers greater than zero**.\
+> **Zero and negative values are not allowed** and will result in an error.
```

Comment:
Why should we throw error on 0?
If someone is auto generating charts, there could be 0 values, which will be valuable to see in the legend. 
Let's throw error on -ve values, and show 0 on the legend.

Reply:
It works like this in the PR:
<img width="568" height="464" alt="image" src="https://github.com/user-attachments/assets/c4be8cf0-42d9-4805-a7e8-849b70c4fc7c" />
The text has been changed to negative numbers. Wording updated